stripecardscan/src/main/java/com/stripe/android/stripecardscan/camera/Camera1Adapter.kt

```diff
@@ -280,11 +324,11 @@ internal class Camera1Adapter(
             val displayMetrics = DisplayMetrics()
             activity.windowManager.defaultDisplay.getRealMetrics(displayMetrics)
 
-            val displayWidth = max(displayMetrics.heightPixels, displayMetrics.widthPixels)
-            val displayHeight = min(displayMetrics.heightPixels, displayMetrics.widthPixels)
+            val previewWidth = max(previewView.height, previewView.width)
+            val previewHeight = min(previewView.height, previewView.width)
 
             val height: Int = minimumResolution.height
-            val width = displayWidth * height / displayHeight
+            val width = previewWidth * height / previewHeight
 
             getOptimalPreviewSize(parameters.supportedPreviewSizes, width, height)?.apply {
```

Comment:
Note: In bouncer's usecase, displayHeight is greater than previewHeight, as it includes the height for android's bottom bar and status bar.  This resulted in the `getOptimalPreviewSize` returning a different value in some cases - 
- before the change the screen bound was (2340, 1080), optimalPreviewSize was (1920, 960)
- after the change the preview bound was (2063, 1080), optimalPreviewSize was (1280, 720)